### PR TITLE
[v7r3] removing env variable DIRAC_USE_NEWTHREADPOOL

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Subsystem
CHANGE: removing env variable DIRAC_USE_NEWTHREADPOOL

ENDRELEASENOTES

### CODE REVIEW

**Comment by chrisburr** [comment]
This also needs to be removed from a few places in `.github/workflows/integration.yml`.

**Review by chrisburr — COMMENTED** [review]
I think these are pre-existing things were you've only changed the formatting but I'll mention them anyway.

File: src/DIRAC/TransformationSystem/Agent/TaskManagerAgentBase.py [hunk 1]
```diff
@@ -271,95 +289,90 @@ def _getClients(self, ownerDN=None, ownerGroup=None):
...
+            JobStatus.MATCHED,
+            JobStatus.RESCHEDULED,
+            JobStatus.COMPLETING,
+            JobStatus.COMPLETED,
+            JobStatus.SUBMITTING,
+            JobStatus.RECEIVED,
+            JobStatus.WAITING,
+            JobStatus.RUNNING,
+            "Scheduled",
+            "Assigned",
```
[the remark this answers is not in the document]
Comment (fstagni) [reply src/DIRAC/TransformationSystem/Agent/TaskManagerAgentBase.py:361]: These 2 are "tasks status" (in the Transformation System), and "Scheduled" and "Assigned" are not jobs states.

File: src/DIRAC/TransformationSystem/Agent/TaskManagerAgentBase.py [hunk 2]
```diff
@@ -141,60 +128,86 @@ def execute(self):
...
+            transformations,
+            owner=owner,
+            ownerGroup=ownerGroup,
+            ownerDN=ownerDN)
 
     # Determine whether the submission of tasks is to be performed
-    enableSubmission = self.am_getOption('SubmitTasks', 'yes')
-    if not enableSubmission:
-      self.log.verbose("Submission of tasks is disabled. To enable it, create the 'SubmitTasks' option")
+    if not self.am_getOption('SubmitTasks', 'yes'):
```
[the remark this answers is not in the document]
Comment (fstagni) [reply src/DIRAC/TransformationSystem/Agent/TaskManagerAgentBase.py:201]: if in the CS there's `no` then the you would still submit tasks. This is the same for all the other options, so not ideal but "historical". Honestly I would not change it in this PR.

File: src/DIRAC/TransformationSystem/Client/Utilities.py [hunk 3]
```diff
@@ -57,28 +57,22 @@ def __init__(self, plugin='Standard', transClient=None, dataManager=None, fc=Non
     self.dmsHelper = DMSHelpers()
 
     self.plugin = plugin
-    self.transID = transID
+    self.transID = str(transID) if transID else 'None'
```
[the remark this answers is not in the document]
Comment (fstagni) [reply src/DIRAC/TransformationSystem/Client/Utilities.py:60]: It's a somewhat "highly specialized" printing here. It signify that the printing is not for a specific TransformationID. TBH I don't so much like it but again I don't want to change it in this PR.

**Review by chrisburr — APPROVED** [review]
The pylint failures should be fixed by https://github.com/DIRACGrid/DIRAC/pull/5170

**Pull request merged**